### PR TITLE
2 project proposal

### PR DESCRIPTION
**Pull request by nymann**
This pull request should contain the project proposal pre-review.

### CODE REVIEW

**Comment by nymann** [comment]
I assigned Simon to be responsible for this, as he is the person that has to send it in the end.

File: project_proposal/tex/mainpage.tex [hunk 1]
```diff
@@ -0,0 +1,390 @@
...
+\begin{itemize}
+    \item This Batch ID
+    \item Product type
+    \item Amount of products (total, defect and acceptable)
+    \item Amount of time used in the different states
+    \item Logging of temperature over the production time
+    \item Logging of humidity over the production time
+\end{itemize}
+
+The MES/SCADA system must be able to monitor the production and display live relevant data from the machine.
```
Comment (Kechr) [inline project_proposal/tex/mainpage.tex]: SCADA = Supervisory control and data acquisition

The MES/SCADA (Supervisory control and data acquisition) system must...

File: project_proposal/tex/mainpage.tex [hunk 2]
```diff
@@ -0,0 +1,390 @@
...
+The documentation of the system must contain an illustration that defines the different components in the setup, in relation to the ISA88\^{}1213 Part 1 Physical Hierarchy model.
+The system must have a visualization that can be accessed and used to display the production data.
+The system must be able to collect the necessary data from the machine and calculate the OEE\^{}131516 of the machine. The OEE must be available to be displayed by the system.
```
Comment (Kechr) [inline project_proposal/tex/mainpage.tex]: OEE = Overall equipment effectiveness

and calculate the overall equipment effectiveness, OEE\^{}131516, of the machine.

**Review by Kechr — CHANGES_REQUESTED** [review]

File: project_proposal/tex/mainpage.tex [hunk 3]
```diff
@@ -0,0 +1,382 @@
+\section{Motivation}
+As students,
+this semester project gives the group a great learning experience as well as a good idea of how such a project would progress.
+During the project, the group will learn how to create and setup an OPC-UA client,
```
Comment (Kechr) [inline project_proposal/tex/mainpage.tex]: setup and Open Platform Communications Unified Architecture, OPC-UA, client, ...

**Review by Kechr — CHANGES_REQUESTED** [review]

**Pull request merged**